build.sbt

```diff
@@ -101,3 +104,11 @@ lazy val root = Project("kettle-test-framework", file("."))
       pushChanges
     )
   )
+
+addCommandAlias(
+  "formatAndCommit",
+  "; clean " +
+    "; fmt " +
+    "; git add -A" +
+    "; git commit --allow-empty -m \"Formatting files before pushing them to remote\""
+)
```

Comment:
This can go into `.sbtrc` as we already have some aliases in there. In that case it can all go into one line like so:-
```
alias formatAndCommit=;clean;fmt;git add -A;git commit --allow-empty -m "Formatting files before pushing them to remote"
```